sambamba/slice.d

```diff
@@ -321,7 +332,13 @@ int slice_main(string[] args) {
         if (args[2] == "*") {
```

Comment:
With new changes, if BED file is provided, this causes access outside array boundaries.